ShopAppTests/App/Account/Controllers/BaseAddressListViewControllerSpec.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  BaseAddressListViewControllerSpec.swift
+//  ShopAppTests
+//
+//  Created by Evgeniy Antonov on 3/2/18.
+//  Copyright © 2018 RubyGarage. All rights reserved.
+//
+
+import Nimble
+import Quick
+import RxSwift
+import ShopApp_Gateway
+
+@testable import ShopApp
+
+class BaseAddressListViewControllerSpec: QuickSpec {
```

Comment:
Add tests for AddressListHeaderViewDelegate, AddressListTableCellDelegate, AccountAddressFormControllerDelegate

Reply:
Method from AddressListHeaderViewDelegate change two variables with fileprivate protection level and call 'performSegue', but BaseAddressListViewController hasn't segues. For now we can't test transition between view controllers